README.md

```diff
@@ -53,81 +20,77 @@ terraform {
 }
 ```
 
-If you're building the provider, follow the instructions to [install it as a plugin.](https://www.terraform.io/docs/plugins/basics.html#installing-a-plugin) After placing it into your plugins directory,  run `terraform init` to initialize it.
-
 ### Setting up provider
-
 ```hcl
-
 provider "thousandeyes" {
   token = "xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx"
 }
 
 ```
 
-The provider requires that the token be set, the token may instead be passed via the environment variable `TE_TOKEN`.
+The provider requires a token. The token can be set on the `token` variable, as shown in the example, or it may instead be passed via the `TE_TOKEN` environment variable.
 
 The provider also supports the following optional settings:
 
 - `account_group_id` may be set to distinguish between affected account groups, if your ThousandEyes account supports more than one.  This may instead be set by the environment variable `TE_AID`.
 - `timeout` may be set to specify the number of seconds to wait for responses from the ThousandEyes endpoints.  This may instead be set by the environment variable `TE_TIMEOUT`.  If this is unset or set to `0`, then the thousandeyes-sdk-go library will use its default settings.
 
-### HTTP Test
+### Examples
+Example of an HTTP test:
 
 ```hcl
-
-data "thousandeyes_agent" "test_agent" {
-  name  = "na-sjc-2-te [VS01]"
+data "thousandeyes_agent" "arg_cordoba" {
+  agent_name = "Cordoba, Argentina"
 }
 
-resource "thousandeyes_http_server" "google_http_test" {
-  name = "google test"
-  interval = 120
-  url = "https://google.com"
-  agents {
-      agent_id = data.thousandeyes_agent.test_agent.agent_id
-  }
-  agents {
+resource "thousandeyes_http_server" "www_thousandeyes_http_test" {
+  test_name      = "Example HTTP test set from Terraform provider"
+  interval       = 120
+  alerts_enabled = false
+
+  url = "https://www.thousandeyes.com"
 
-      agent_id = 12345
+  agents {
+    agent_id = data.thousandeyes_agent.arg_cordoba.agent_id
   }
 }
 ```
 
-### Agent to Server
+### Supported tests
+- [X] agent-to-agent
+- [X] agent-to-server
+- [X] alert-rule
+- [X] bgp
+- [X] dnssec
+- [X] dns-server
+- [X] dns-trace
+- [X] ftp-server
+- [X] http-server
+- [X] page-load
+- [X] sip-server
+- [ ] transactions
```

Comment:
Are this the classical transaction test? If so they were already deprecated

Reply:
Good catch!